src/Bicep.Cli/Commands/BuildCommand.cs

```diff
@@ -46,8 +49,22 @@ public async Task<int> RunAsync(BuildArguments args)
                 logger.LogWarning(CliResources.ResourceTypesDisclaimerMessage);
             }
 
+
             if (!IsBicepFile(inputPath))
             {
+                
+                if(IsBicepparamsFile(inputPath))
```

Comment:
Any ideas on doing this differently? Also should I worry about stdout for params right now?

Reply:
That would just mean wrapping what you have here in the following, right?

```csharp
if (args.OutputToStdOut)
{
    writer.ToStdout(compilation);
} else {
    // current implementation
}
```

Reply:
yes, I do think there is already a lot of duplication going on in ```BuildCommand```